lib/xclarity_client/xclarity_resource.rb

```diff
@@ -4,6 +4,7 @@ def build_resource(attributes)
       attributes.each do |key, value|
         begin
           value = value.gsub("\u0000", '') if value.is_a?(String)
+          key = key.gsub("-","_")
```

Comment:
@MaysaMacedo Just wondering about the background regarding this change. What change was made in LXCA that made this necessary? Just curious. Thanks